README.md

```diff
@@ -88,7 +93,8 @@ The `values.yaml` should be a locally kept yaml with the user's environment spec
 ```yaml
 superblocks:
   agentKey: <agent-key> # obtained during agent onboarding
-  agentHostUrl: "http[s]://<agent-host[:port]>/agent"
```

Comment:
We do need to have double quotes for `SUPERBLOCKS_AGENT_ENVIRONMENT="*"`, otherwise `zsh` tries to expand the `*`. But do need double quotes for `values.yaml`?

Reply:
I stay corrected - we do need double quotes in the yaml file as well:
```
~  cat >values.yaml <<EOL

containerPort: 9999

superblocks:
  agentKey: dev-agent-key
  agentHostUrl: http://localhost:9999/agent
  agentEnvironment: *
EOL
☁  ~  helm upgrade superblocks-agent superblocks/superblocks-agent \
  --install \
  --namespace superblocks \
  --create-namespace \
  -f values.yaml
Release "superblocks-agent" does not exist. Installing it now.
Error: failed to parse values.yaml: error converting YAML to JSON: yaml: line 7: did not find expected alphabetic or numeric character
☁  ~  cat >values.yaml <<EOL

containerPort: 9999

superblocks:
  agentKey: dev-agent-key
  agentHostUrl: http://localhost:9999/agent
  agentEnvironment: "*"
EOL
☁  ~  vi values.yaml
☁  ~  helm upgrade superblocks-agent superblocks/superblocks-agent \
  --install \
  --namespace superblocks \
  --create-namespace \
  -f values.yaml
Release "superblocks-agent" does not exist. Installing it now.
NAME: superblocks-agent
LAST DEPLOYED: Fri Feb 25 17:54:43 2022
NAMESPACE: superblocks
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
1. Get the application URL by running these commands:
  export POD_NAME=$(kubectl get pods --namespace superblocks -l "app.kubernetes.io/name=superblocks-agent,app.kubernetes.io/instance=superblocks-agent" -o jsonpath="{.items[0].metadata.name}")
  export CONTAINER_PORT=$(kubectl get pod --namespace superblocks $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
  echo "Visit http://127.0.0.1:8080 to use your application"
  kubectl --namespace superblocks port-forward $POD_NAME 8080:$CONTAINER_PORT
```